Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.8.5-slim-bookworm
+FROM python:3.10.13-slim-bookworm
```

Comment:
This should be tested a bit more throughly since the media service tests are not very good

Reply:
Would the `Dockerfile.dev` being on python3.10 since 2022-07-04 count as thoroughly tested for you?

Reply:
> Would the `Dockerfile.dev` being on python3.10 since 2022-07-04 count as thoroughly tested for you?

I'm not sure, since I don't know how often people work with files in their local dev setup. I, for my part, don't do that very often.

> Seems the production image was out of sync for some time now ..

Yes, I figured. But better be save than sorry - we should just up- and download a few files after the staging update to make sure everything works.